### ISSUE TITLE
Feature Request - Movement dials on printouts

### ISSUE THREAD

**Issue by cwoody0** [geordanr/xwing#450]
Hello,
I use the heck out of, and love, (Yet Another) X-Wing Squad Builder, and I build all my lists there. However, whenever I go to actually play, I build the list in http://xwing-builder.co.uk/build before printing it. The reason is that one has the option to include the movement dials on the printout. If you could include an option to have the movement dials on your printouts, you'd fix all that is wrong with this world.
Thanks!

**Comment by geordanr** [comment]
Good idea; I need to figure out how to change the layout of the printout so things still look good (and then find someone who's actually good at CSS to help make it happen).  I'll see what I can do.
